src/k8s-extension/azext_k8s_extension/_params.py

```diff
@@ -52,30 +52,41 @@ def load_arguments(self, _):
                    help='Specify the release train for the extension type.')
         c.argument('configuration_settings',
                    arg_group="Configuration",
-                   options_list=['--configuration-settings', '--config-settings', '--config'],
```

Comment:
>--config-settings [](http://example.com/codeflow?start=62&length=17)

Why is this option removed?
Comment applicable in other places as well.

Reply:
Currently there is a bug in our cli where the formatting of the help arguments is in a way that it becomes unreadable. 
I found out that the problem is the long argument names that are causing the formatting to become unreadable after debugging with @NarayanThiru and anubhav. This would format the help section that is consistent with the rest of az cli.